modular_skyrat/modules/synths/code/surgery/robot_healing.dm

```diff
@@ -84,7 +87,7 @@
 
 /datum/surgery_step/robot_heal/initiate(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery, try_to_fail = FALSE)
 	if(..())
-		while((heals_brute && target.getBruteLoss() && tool.tool_use_check(user, 1)) || (heals_burn && target.getFireLoss() && tool))
+		while((heals_brute && target.getBruteLoss() && tool/*.tool_use_check(user, 1)*/) || (heals_burn && target.getFireLoss() && tool))
```

Comment:
This is in a modular file-- I don't see the need to comment it out.

Reply:
OH! I forgot to remove that part, I'll handle it now.